### PR TITLE
Deven Mukkamala HW01

### PR DESCRIPTION
**Pull request by dmukkamala**
I forked the repository, cloned it, and opened it in R by using the url. I am new to R, Git, and Github, so I was way too cautious with everything. When it came to doing the actual biography in Markdown, I had no trouble. The guide was really easy to interpret, and I only had a little trouble with the formatting of the lists. The process of using the commit function was relatively easy, but it took a couple of attempts. 

All in all, I think the assignment made me more comfortable with the different applications and removed some of my caution. I used a few outside resources, including some YouTube videos, but they didn't really help. Re-reading the instructions was what eventually cleared up some issues.

### CODE REVIEW

**Comment by claireanastasia** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |    x       |              |            |

Remarks:

* Elaborate on above, especially for "needs work."
I thought your bio seemed excellent! You included several of the different tools in Markdown including bold, italics, a picture, headers, and links, so it seems like you went above and beyond the assignment requirements!
* Some specific praise?
I think the presentation worked out very well. The picture was a good size and positioned in a visually pleasing way. 
* Something I learned?
I used asterisks to emphasize the text in my bio. Looking at your bio showed me that I could also have used underscores to italicize my text. 
* Specific constructive criticism?
Going forward it might be helpful to give your commits different names, but this assignment is simple enough that I don't think it matters. 
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
I am new to R, Git Hub, and Git, too. I agree that the class instructions ended up being the most helpful. One site I used that may be helpful going forward is: https://www.rstudio.com/resources/cheatsheets/

**Review by iradner — COMMENTED** [review]
## Peer Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

This is a nice clean and easy to understand code that I was able to run on my own computer (aka it was reproducible) and it seems like the pull request, etc. on git hub was  all executed correctly (using git / git hub was the hardest part for me as well)! 

The picture was a good touch. That part was the hardest for me in my bio, and it seemed like you didn't have a problem with it. And good thinking using an online url. I initially pasted in the path to an image that was stored locally on my computer before realizing that only I would be able to see it then!

I am not sure how you ended up navigating the git commands in R, but I ended up using the commands in the terminal on R instead of the git GUI in R. Also, this webpage was really helpful for me: http://r-pkgs.had.co.nz/git.html

**Comment by yeonjang** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |             |

Remarks:

* Elaborate on above, especially for "needs work."  
Efficient and clear code. Used all of the different tools. Nice organization. 
* Some specific praise?
Good bio. Good use of different tools throughout. 
* Specific constructive criticism?
The commits are a bit unclear what happened.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+A few of my hobbies of mine are: 
+
+* Basketball (The Chicago Bulls)
+* Reading (Non-fiction)
+* French Horn 
+* Debate
+
+If I had to rank my favorite philosophers based on _writing style_, rather than _beliefs_, I would say:
+
+1. Jean-Jacques Rousseau
```
Comment (bensoltoff) [inline README.md:17]: You don't have to increment the numbers yourself. When the document renders, this automatically happens.

```
1. Jean-Jacques Rousseau
1. Thomas Hobbes 
1. Plato
```

becomes

1. Jean-Jacques Rousseau
1. Thomas Hobbes 
1. Plato

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 01
+# HW01
+## Deven Mukkamala, Biography
+
+![Deven Mukkamala](https://www.legion.org/sites/legion.org/files/legion/boys-nation-delegates/20170721_Michigan%20Federalist%20Deven%20Mukkamala_7501.jpg)
```
Comment (bensoltoff) [inline README.md:4]: Great use of embedding a photo from an external site. If it wasn't hosted on another web page (for instance, something just on your computer), you could put the image in your Git repo and embed it from there.

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |        x      |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |           |        x      |            |
| **Ease of access**          |    x       |              |            |

Remarks:

* Nice variety of Markdown syntax

**Pull request closed without merging**